src/linalg/dense/ParELAG_MultiVector.cpp

```diff
@@ -361,7 +357,7 @@ MultiVector::InversePermutation(const int * p, int p_size, int out_size) const
     int offset_in = 0;
 
     if(out_size > p_size)
-        std::fill(out->data, out->data+ out->size, 0.0);
+        std::fill((double *)out->data, (double *)out->data + out->size, 0.0);
```

Comment:
In this case I think I might want an explicit loop over elements rather than `std::fill` with the cast (who knows what `std::fill` will do with GPU/device memory?)

Reply:
Done